app/models/course/discussion/topic.rb

```diff
@@ -77,4 +77,8 @@ def unmark_as_pending
     self.pending_staff_reply = false
     save
   end
+
+  def first_poster
```

Comment:
Just a random question, is there a case where the `first_poster` is different from the topic `creator`?

Reply:
Neither `Discussion::Topic` nor `Video::Topic` has a creator field actually. So I'm using the first poster here since they are created together.

Reply:
Any views on whether we should include a creator field for the `Video::Topic`?

Reply:
Actually, sure, I don't see why not now that each topic is a single thread. Would you prefer if I do that in another PR? I can edit this one afterwards.

Reply:
Ok either way. This PR is not too big, I don't foresee it expanding the PR a lot. Should still be easy to review.

Reply:
Alright, I'll add it here then, thanks!